kinto/__main__.py

```diff
@@ -141,6 +157,14 @@ def main(args=None):
                                          parsed_args['bucket'],
                                          parsed_args['collection'])
 
+    elif which_command == 'revert':
+        env = bootstrap(config_file)
+        return scripts.revert(env,
+                              auth=parsed_args['auth'],
+                              bucket=parsed_args['bucket'],
+                              collection=parsed_args['collection'],
+                              since=parsed_args['since'])
```

Comment:
If we follow the *URL endpoint* approach (which is better indeed), then the command-line is probably useless.